Sources/Networking/Responses/CustomerInfoResponse.swift

```diff
@@ -100,6 +105,12 @@ extension CustomerInfoResponse {
 
     }
 
+    #if ENABLE_VIRTUAL_CURRENCIES
+    internal struct VirtualCurrencyInfo {
+        let amount: Int
```

Comment:
This may change to `balance` in the near future, I'll update this here or in a follow-up PR, depending on when/if the changes are made